src/main/resources/assets/skyblocker/lang/en_us.json

```diff
@@ -29,6 +29,9 @@
 
   "skyblocker.config.crimsonIsle": "Crimson Isle",
 
+  "skyblocker.config.crimsonIsle.extendNetherFog": "Extend Nether Fog",
+  "skyblocker.config.crimsonIsle.extendNetherFog.@Tooltip": "When in the Crimson Isles, prevents fog from appearing closer to the player as it does in vanilla nether biomes",
```

Comment:
Isles shouldn't be plural and the sentence should end with a period

Reply:
TIL I've been saying Crimson Isle wrong for 2 years lol